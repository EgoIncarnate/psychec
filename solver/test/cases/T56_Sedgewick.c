// THIS IS NOT THE FULL CODE
// Sedgewick chap 17, example 10
// tree example 

void STinit()
{ head = NULL; N = 0; }
link stNEW(int d)
{ link x = malloc(sizeof *x);
  x->index = -1; x->d = d;
  x->l = NULL; x->m = NULL; x->r = NULL;
  return x;
}
