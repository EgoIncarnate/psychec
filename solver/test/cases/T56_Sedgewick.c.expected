typedef  struct __link__ {int index;int d;uintptr_t* r;uintptr_t* m;uintptr_t* l;}* link ;
uintptr_t N ;
uintptr_t* head ;
